gateway-messages/tests/versioning/v6.rs

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The tests in this module check that the serialized form of messages from MGS
+//! protocol version 5 have not changed.
+//!
+//! If a test in this module fails, _do not change the test_! This means you
+//! have changed, deleted, or reordered an existing message type or enum
+//! variant, and you should revert that change. This will remain true until we
+//! bump the `version::MIN` to a value higher than 4, at which point these tests
+//! can be removed as we will stop supporting v5.
```

Comment:
```suggestion
//! bump the `version::MIN` to a value higher than 6, at which point these tests
//! can be removed as we will stop supporting v6.
```

(also, if you wouldn't mind fixing the block comment in `v5.rs`, it's also wrong)

Reply:
Fixed in https://github.com/oxidecomputer/management-gateway-service/pull/92/commits/1879755cbe398073368338d11717a8ac8dc7fd93